nautobot/extras/migrations/0092_uniqueness_data_migration.py

```diff
@@ -3,11 +3,13 @@
 from nautobot.core.utils.migrations import check_for_duplicates_with_natural_key_fields_in_migration
 
 
-def ensure_unique_scheduledjob_names(apps, schema_editor):
+def ensure_unique_scheduledjob_names_and_save_job_class_parameter_to_task(apps, schema_editor):
     ScheduledJob = apps.get_model("extras", "ScheduledJob")
 
     cache = set()
     for instance in ScheduledJob.objects.all():
+        instance.task = instance.job_class
+        instance.save()
```

Comment:
Was there a reason this couldn't/shouldn't be done in 0091 (where we're updating `job_class` to 2.0 standards) instead? This is probably fine, just wondering.

Reply:
Ur right, it makes more sense to include it in 0091.